src/components/Subscriptions.js

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useEffect } from 'react'
+import { connect } from 'react-redux'
+import { setLastLocation } from '../actions/setLastLocationAction'
+import { Header, Container, Segment, Grid } from 'semantic-ui-react'
+import PayPalAgreementNew from './PayPalAgreementNew'
+import createBillingAgreement from '../helpers/createBillingAgreement'
+import queryString from 'query-string'
+import '../assets/Subscriptions.css'
+
+export const Subscriptions = props => {
+  useEffect(() => {
```

Comment:
I tried to access the URL for this component directly but I got a blank page

Reply:
Although, maybe like the agileventures site, we should have `subscriptions/new` default to premium sign up info